test-integration/repay-loan/1_repay_loan_3_payments.js

```diff
@@ -0,0 +1,190 @@
+// Util classes
```

Comment:
Please, remove filename to 1_repay_loan_eth_3_payments.js (or similar). So we know it uses ETH as collateral.

Reply:
renamed